build_and_setup_everything_bazel_manually.sh

```diff
@@ -16,5 +16,7 @@
 
 . ./grant_permissions.sh
 
+kubectl patch --type=merge StorageProfile standard -p '{"spec":{"claimPropertySets":[{"accessModes":["ReadWriteOnce"],"volumeMode":"Filesystem"}]}}'
```

Comment:
we can drop that when #12 gets in, right?

Reply:
Yes, it is the same as ovirt/patch.yml from that PR.